core/templates/dev/head/base.html

```diff
@@ -255,10 +255,20 @@
                           <ul ng-if="!windowIsNarrow" class="nav oppia-navbar-tabs">
                             <create-activity-button></create-activity-button>
 
-                            <li class="oppia-clickable-navbar-element pull-right">
-                              <a class="oppia-navbar-tab" href="/donate" translate="I18N_TOPNAV_DONATE">
+                          <ul ng-if="windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
```

Comment:
But that's intentional -- and even with the current code, the other buttons disappear on a small screen.

What I was trying to say is that the HTML is incorrectly formed. There's an extra opening 'ul' tag that isn't closed properly, and the indentation should be two spaces forward from where it currently is. That said, there's already a special case for narrow windows (lines 252-254) so the extra ng-if here is unnecessary. I played around a bit with it and came up with the following; would something like this work instead?

```
  <ul ng-if="windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
    <create-activity-button></create-activity-button>
  </ul>
  <ul ng-if="!windowIsNarrow" class="nav oppia-navbar-tabs">
    <create-activity-button></create-activity-button>
    <li>
      <div class="pull-right oppia-navbar-button-container">
        <a href="/donate" class="btn oppia-navbar-button oppia-navbar-hide-on-small-width protractor-test-create-activity oppia-transition-200"
          <span translate="I18N_TOPNAV_DONATE"></span>
        </a>
      </div>
    </li>
  ...
```